doc/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ if(${WEST} STREQUAL WEST-NOTFOUND)
   unset(WEST)
 endif()
 
-set(ZEPHYR_BASE $ENV{ZEPHYR_BASE})
+file(TO_CMAKE_PATH $ENV{ZEPHYR_BASE} ZEPHYR_BASE)
```

Comment:
We should use the same mechanism as in a normal build. So either apply this change also the normal build, or use the below mechanism in the doc build:

string(REPLACE "\\" "/" ZEPHYR_BASE ${ZEPHYR_BASE})

Reply:
I will apply the same mechanism to the normal build then

Reply:
Fixed